scss/_base_typography-max-widths.scss

```diff
@@ -1,3 +1,16 @@
+@mixin vf-b-typography-max-widths {
+  %max-width--p {
+    max-width: map-get($max-widths, default);
+  }
+
+  @include deprecate('3.0.0', 'Use `%max-width--p` instead') {
```

Comment:
I still don't understand the need for these new controls that do the same as the `p-max-width` mixin?

Reply:
These aren't new, they were in 2.0.0 but are now removed from master, making it a breaking change. Reintroducing them and marking them as deprecated is the most responsible way of supporting the API.